README.md

```diff
@@ -30,20 +42,46 @@ cargo dsync -i schema.rs -o models
 Now we have everything we need!
 
 ```rust
-use models::todos;
```

Comment:
why remove that line? i think this would make sense to get to know where some things come from